### PR TITLE
Images Positioning

### PR DESCRIPTION
**Pull request by vrezhMikTumo**
Images were appearing in a different horizontal positions. Objects that supposed to be in the middle were appearing a bit left or right. ImagesInVuewBoard calculates their sizes and add deviation so the objects appear in the closest position they supposed to be in each frame.

### CODE REVIEW

File: src/components/ImagesInViewBoard.vue [hunk 1]
```diff
@@ -11,14 +11,20 @@ const props = defineProps(["boardId", "objectId"]);
 const object = computed(
   () => Store.boards[props.boardId].objects[props.objectId]
 );
+console.log(object.value);
```
Comment (tumochristine) [inline src/components/ImagesInViewBoard.vue:14]: remove console log

File: src/components/ImageObject.vue [hunk 2]
```diff
@@ -46,6 +46,7 @@ img {
 }
```
Comment (tumochristine) [inline src/components/ImageObject.vue:46]: In the commit message, can you clarify which view this is for? Is this the edit board or the preview board or something else?

**Review by tumochristine — CHANGES_REQUESTED** [review]

File: src/components/ImagesInViewBoard.vue (hunk 1 above)
Comment (vrezhMikTumo) [reply src/components/ImagesInViewBoard.vue:14]: done

File: src/components/ImageObject.vue (hunk 2 above)
Comment (vrezhMikTumo) [reply src/components/ImageObject.vue:46]: done

**Review by tumochristine — APPROVED** [review]

**Pull request merged**